### PR TITLE
sync logger

### PR DESCRIPTION
**Pull request by almogdepaz**
use gopkg.in/op/go-logging.v1 to log in sync, mesh and blockListener

### CODE REVIEW

File: mesh/mesh.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package mesh
 import (
 	"errors"
 	"github.com/spacemeshos/go-spacemesh/database"
-	"github.com/spacemeshos/go-spacemesh/log"
+	"gopkg.in/op/go-logging.v1"
```
Comment (y0sher) [inline mesh/mesh.go]: why not use `log.Log`?

Comment (almogdepaz) [reply mesh/mesh.go]: thanks

**Review by antonlerner — APPROVED** [review]

File: sync/syncer_test.go [hunk 2]
```diff
@@ -344,7 +336,7 @@ func Test_TwoNodes_SyncIntegrationSuite(t *testing.T) {
 	i := 1
 	sis.BeforeHook = func(idx int, s p2p.NodeTestInstance) {
 		l := log.New(fmt.Sprintf("%s_%d", sis.name, i), "", "")
```
Comment (y0sher) [inline sync/syncer_test.go:338]: the second and third args of the new methods choose in which file this log will be saved. left empty the log isn't save to file.
The best practice (in my opinion) will be to create the log at the most top level and pass it down to each component so all the logs can be centered in one file.

Comment (y0sher) [reply sync/syncer_test.go:338]: currently log is initialized in the p2p layer so you wouldn't be able to do that. I'll open an issue about it.

**Pull request merged**